src/ui/components/NetworkMonitor/RequestRow.tsx

```diff
@@ -22,7 +22,7 @@ export const RequestRow = ({
   isInLoadedRegion: boolean;
   isInPast: boolean;
   isSelected: boolean;
-  onClick: (row: RequestSummary) => void;
+  onClick: (row: RequestSummary, event: React.MouseEvent) => void;
```

Comment:
This change still seems unnecessary. Nothing is using the event param, is it?

Reply:
Mistake, removed